### PR TITLE
LEAN-1330: EI25Q1-030 - Surface non-editable DOI in Editor

### PR DESCRIPTION
**Pull request by mvidtman**

### CODE REVIEW

File: src/schema/nodes/affiliations.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ import { NodeSpec } from 'prosemirror-model'
 // It exists for the purpose of styling in the UI
 
 export const affiliations: NodeSpec = {
-  content: 'affiliation*',
+  content: 'affiliation* paragraph*',
```
Comment (mbartenev-atypon) [inline src/schema/nodes/affiliations.ts:23]: this change should be only a change in a view of the node, why do we need to change the schema?

File: src/schema/nodes/manuscript.ts [hunk 2]
```diff
@@ -48,11 +50,11 @@ export const manuscript: NodeSpec = {
   ],
   toDOM: (node) => {
     const manuscriptNode = node as ActualManuscriptNode
-
     return [
       'article',
       {
         id: manuscriptNode.attrs.id,
+        doi: manuscriptNode.attrs.doi,
```
Comment (mbartenev-atypon) [inline src/schema/nodes/manuscript.ts:57]: do we need that in the DOM attributes?

**Review by mbartenev-atypon — CHANGES_REQUESTED** [review]

File: src/schema/nodes/affiliations.ts (hunk 1 above)
Comment (mvidtman) [reply src/schema/nodes/affiliations.ts]: this change is reverted....

File: src/schema/nodes/manuscript.ts (hunk 2 above)
Comment (mvidtman) [reply src/schema/nodes/manuscript.ts]: this change is reverted....

**Review by mbartenev-atypon — APPROVED** [review]

**Pull request merged**